src/useKeybind.ts

```diff
@@ -3,7 +3,7 @@ import { useEffect, useRef } from "react";
 type Modifiers = ("ctrl" | "alt" | "shift")[];
 type EventType = "keyup" | "keydown";
 
-interface KeybindArgs {
```

Comment:
why is this exported? its just there to make the args cleaner to type in the codebase.

Reply:
meant to leave the review on the other side of the diff but oh well

Reply:
So TS actually yells at you thay that interface is "private" if you don't export it. It's rhe first I've ever seen it so I just defided to export it instead of messing with your tsconfig because I googled it and it's caused by the "declaration "settings.

Reply:
hmm ok, fair